acto/lib/operator_config.py

```diff
@@ -45,7 +45,7 @@ class DeployConfig(BaseModel, extra="forbid"):
     """Configuration for deploying the operator"""
     steps: List[DeployStep] = Field(
         description="Steps to deploy the operator",
-        min_length=1,)
+        min_items=1,)
```

Comment:
Can you try to upgrade the pydantic version to the version in the pyproject yaml and try again? I think the min_items is deprecated in the old version, and the min_length is the one to go with

Reply:
I see. You are right. `min_length` works after I upgraded the pydantic version.